apps/tome-api/src/features/card/card.fns.ts

```diff
@@ -750,6 +775,173 @@ export const deck: DbCard[] = [
+		type: 'spell',
+		colors: ['blue'],
+		attack: 8,
+		description: 'After a successful attack, you *may* (NOT IMPLEMENTED) send a RED or NEUTRAL [FIELD] spell to the discard pile.',
+		// TODO: confirmation of "activate <spellname> effect?" for cards worded MAY
+		effects: {
+			onDealDamage: async function* ({ actions, game, ownerSide }) {
+				const fieldToDiscard = topOf(game.board.field);
+				if (fieldToDiscard?.color === 'red' || fieldToDiscard?.color === null) {
+					yield* actions.discard({ card: fieldToDiscard, from: game.board.field, side: ownerSide });
+				}
+			},
+		},
+	},
+	{
+		id: '60',
+		name: 'Parched Earth',
+		type: 'spell',
+		colors: ['green'],
+		attack: 8,
+		description: 'After a successful attack, you *may* (NOT IMPLEMENTED) send a BLUE or NEUTRAL [FIELD] spell to the discard pile.',
+		effects: {
+			onDealDamage: async function* ({ actions, game, ownerSide }) {
+				const fieldToDiscard = topOf(game.board.field);
+				if (fieldToDiscard?.color === 'blue' || fieldToDiscard?.color === null) {
+					yield* actions.discard({ card: fieldToDiscard, from: game.board.field, side: ownerSide });
+				}
+			},
+		},
+	},
+	{
+		id: '61',
+		name: 'Wobbly bridge',
+		type: 'field',
+		color: 'green',
+		description: 'Spells deal +2X damage during combat, where X is the size of the field effect stack',
+		effects: {
+			beforeCombat: async function* ({ turn, game }) {
+				const fieldEffectStack = game.board.field.length;
+				for (const combat of turn.combatStack) {
+					if (!combat.source) continue;
+					if (combat.type !== 'damage') continue;
+					if (combat.source.type !== 'spell') continue;
+					combat.value += 2 * fieldEffectStack;
+				}
+				yield game;
+			},
+		},
+	},
+	{
+		id: '62',
+		name: 'Mirror shield',
+		type: 'spell',
+		colors: [],
+		attack: 0,
+		description:
+			'If this spell is beaten in combat, the damage that would have been caused to you is caused to the opponent instead. If this spell is not beaten in combat, discard it.',
+		effects: {
+			onClashWin: async function* ({ actions, game, thisCard, ownerSide }) {
+				yield* actions.discard({ card: thisCard, from: game.board.field, side: ownerSide });
+			},
+			onClashLose: async function* ({ game, ownerSide, opponentSide }) {
+				for (const combat of game.turn.combatStack) {
+					if (combat.type !== 'damage') continue;
+					if (combat.target !== ownerSide) continue;
+					combat.target = opponentSide;
+				}
+			},
+		},
+	},
+	{
+		id: '63',
+		name: 'Cannon Fire',
+		type: 'spell',
+		colors: ['red'],
+		attack: 25,
+		description:
+			'When this spell is used in combat, select a card from your hand and discard it. If you don’t have cards to discard, discard Cannon Fire instead.',
+		effects: {
+			beforeCombat: async function* ({ game, actions, ownerSide, thisCard }) {
+				if (game.board.players[ownerSide].hand.length === 0) {
+					yield* actions.discard({ card: thisCard, from: game.board.field, side: ownerSide });
```

Comment:
this is a spell, you gotta use the stack this card is! (we probably need a util "getSpellStack")
on a side node, perhaps we declare _after_ the damage (like refueling a cannon)? 
the reason is that this will sometimes be frustrating to the user: they wanna attack with cannon fire but they'll instead discard it and attack with what's underneath accidentally.

Reply:
Good point!